### PR TITLE
[ALLUXIO-3294] Remove lineage

### PR DESCRIPTION
**Pull request by aaudiber**
https://alluxio.atlassian.net/browse/ALLUXIO-3294

After this change, the only lineage-related code is
the handling for lineage journal entries. This is so
that users can upgrade from a version where they
tried out lineage.

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java [hunk 1]
```diff
@@ -248,7 +248,7 @@ public void stop() throws Exception {
   }
 
   /**
-   * Starts all masters, including block master, FileSystem master, lineage master and additional
+   * Starts all masters, including block master, FileSystem master, and additional
```
[the remark this answers is not in the document]
Comment (aaudiber) [reply core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java]: fixed, thanks

**Comment by apc999** [comment]
- remove Lineage doc?

**Comment by aaudiber** [comment]
@apc999 Which docs are you thinking of?

**Comment by apc999** [comment]
yay!

**Pull request merged**